src/sass/_shame-wizard.scss

```diff
@@ -0,0 +1,225 @@
+// This file is a place where you put all the code you're not proud of,
```

Comment:
Hehe, 'ware the _shame-wizard! He'll replay awkward teenage years again and again and again and...

Maybe rename this to `src/sass/modules/_m-education-wizard.scss`? It's perhaps not the _best_ place for it, but...